prime-router/metadata/HL7/catchall/hl7/datatypes/DR/Period.yml

```diff
@@ -9,7 +9,7 @@ start:
     dateTimeIn: $start
 _start:
   expressionType: nested
-  condition: $dateTimeIn NOT_NULL
+  condition: $start NOT_NULL
```

Comment:
This mapping was previously not functioning as intended due to referencing an out of scope variable.